src/openvpn/buffer.c

```diff
@@ -442,6 +442,7 @@ format_hex_ex (const uint8_t *data, int size, int maxoutput,
 				    ((size * 2) + (size / (space_break_flags & FHE_SPACE_BREAK_MASK)) * (int) strlen (separator) + 2),
 				    gc);
   int i;
+  ASSERT( maxoutput > 0 || separator != NULL );
```

Comment:
"int i;" always been there

hmm, we can run cppcheck in C89 mode to make sure code is MSVC compliant 
thought ?